torch/testing/_asserts.py

```diff
@@ -140,8 +209,10 @@ def _check_attributes_equal(
             same :attr:`~torch.Tensor.device`.
         check_dtype (bool): If ``True`` (default), checks that both :attr:`actual` and :attr:`expected` have the same
             ``dtype``.
-        check_stride (bool): If ``True`` (default), checks that both :attr:`actual` and :attr:`expected` have the same
-            stride.
+        check_stride (bool): If ``True`` (default) and the tensors are strided, checks that both :attr:`actual` and
+            :attr:`expected` have the same stride.
+        check_is_coalesced (bool): If ``True`` (default) and the tensors are sparse COO, checks that both
+            :attr:`actual` and :attr:`expected` are either coalesced or not-coalesced.
```

Comment:
Are the tensors coalesced before being compared if this check is disabled?

Reply:
Yes, but not in this function. It happens in `_equalize_attributes `. See L267-L270. 

The docstring of the outward facing `assert_close` mentions this behavior. See L856-L858

Reply:
I noticed; if it's sufficient to document it in the assert_close docstring then this is OK